drivers/interrupt_controller/Kconfig.stm32

```diff
@@ -13,31 +13,31 @@ config EXTI_STM32
 	help
 	 Enable EXTI driver for STM32 line of MCUs
 
+
 config EXTI_STM32_EXTI1_0_IRQ_PRI
 	int "EXTI1:0 IRQ priority"
 	depends on EXTI_STM32
-	depends on SOC_SERIES_STM32F0X
```

Comment:
Should be:
depends on SOC_SERIES_STM32F0X || SOC_SERIES_STM32L0X
as 3_2 and 15_4

Reply:
fixed.